src/main/resources/swagger/api-docs.yaml

```diff
@@ -8626,6 +8626,10 @@ components:
           type: boolean
           default: false
           description: Whether or not to create output columns if they would be empty and optional.  Defaults to false (columns are created).
+        perWorkflowCostCap:
+          type: number
+          description: The maximum cost for a single workflow in this submission.  If the workflow exceeds this cost, it will be aborted.
```

Comment:
Didn't we have quibbles over the terminology "maximum" since it might actually exceed this value?